packages/management/framework/package.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: framework.tanzu.vmware.com
+  namespace: framework
+spec:
+  refName: framework.tanzu.vmware.com
+  version: VERSION
+  releaseNotes: "relevant release notes..."
```

Comment:
Should we omit this key if we don't have any release notes here? (I don't know if this gets replaced at runtime)